airbyte-integrations/base/base.sh

```diff
@@ -48,8 +48,11 @@ function main() {
     eval "$AIRBYTE_DISCOVER_CMD" --config "$CONFIG_FILE"
     ;;
   read)
-    # todo: state should be optional: --state "$STATE_FILE"
-    eval "$AIRBYTE_READ_CMD" --config "$CONFIG_FILE" --catalog "$CATALOG_FILE"
+    if [[ -z "$STATE_FILE" ]]; then
```

Comment:
can you use a READ_OPTS that is empty or contains the state file instead? so you dry the eval?